perf/sds-tests/citadel-1/setup_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+NAMESPACE=${NAMESPACE:?"specify the namespace for running the test"}
+NUM=${NUM:?"specify the number of httpbin and sleep workloads"}
+
+function inject_workload() {
+  local deployfile="${1:?"please specify the workload deployment file"}"
+  # This test uses perf/istio/values-istio-sds-auth.yaml, in which
+  # Istio auto sidecar injector is not enabled.
+  istioctl kube-inject -f "${deployfile}" -o temp-workload-injected.yaml
```

Comment:
please download the correct version istioctl here.

Reply:
Done.